frameworks/youth-risk-assessment/questions/current-behaviour.yml

```diff
@@ -0,0 +1,9 @@
+type: textarea
+question: Give details of their current behaviour.
```

Comment:
No need for the fullstop at the end here. We haven't used that in any other question fields as far as I'm aware?

Reply:
I also wonder if we should follow a more consistent pattern and have this is a "question": `What is their current behaviour?`

Reply:
Don't need full stop. I think we had a reason for now saying 'What...' - but I can't remember now. In theory, I'm not against - sounds better - I think there's a reason though... something to discuss